<?xml version="1.0" ?>
<launch>
    <node pkg="kinect2" type="kinect2_node" name="kinect2" output="screen" >
        <param name="topic_color" value="/kinect2/image_color"/>
        <param name="topic_color_info" value="/kinect2/image_color/camera_info"/>
        <param name="topic_depth" value="/kinect2/depth"/>
        <param name="topic_depth_info" value="kinect2/depth/camera_info"/>
        <param name="publish_color_registered" value="true"/>
        <param name="publish_color" value="true"/>
        <param name="publish_ir" value="false"/>
        <param name="publish_depth" value="false"/>
    </node>
</launch>

